### PR TITLE
Document Job status and isUserCancelled metrics

### PR DESCRIPTION
**Pull request by shultseva**
https://hazelcast.atlassian.net/browse/HZ-3665

### CODE REVIEW

File: docs/modules/ROOT/pages/list-of-metrics.adoc [hunk 1]
```diff
@@ -92,6 +92,21 @@ execution.
...
+0 - submitted;
+1 - initialization phase;
+2 - running;
+3 - suspended;
+4 - exporting snapshot;
+5 - completing phase;
+6 - failed;
+7 - completed successfully;
+
+|userCancelled
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/list-of-metrics.adoc:106]: `userCancelled` name seems to be in UK English (two Ls in cancelled), description in US (single L in canceled).

Not really a docs thing if the UK English is used in the code, but thought I'd mention it....

Comment (Serdaro) [reply docs/modules/ROOT/pages/list-of-metrics.adoc:106]: See https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/jet/core/metrics/MetricNames.java#L212

**Review by Serdaro — APPROVED** [review]

**Comment by Serdaro** [comment]
It'd be nice to standardize the formatting when listing the potential values.

**Comment by rebekah-lawrence** [comment]
> It'd be nice to standardize the formatting when listing the potential values.

I'd like to see the wording change to "valid values" as the standard; but again, that's a docs thing and nothing to do with this PR ;-)

**Comment by Serdaro** [comment]
> > It'd be nice to standardize the formatting when listing the potential values.
> 
> I'd like to see the wording change to "valid values" as the standard; but again, that's a docs thing and nothing to do with this PR ;-)

This has been a wound for so long, maybe let's create a ticket on standardizing the listings. There are "available"s, "valid"s, etc.

File: docs/modules/ROOT/pages/list-of-metrics.adoc (hunk 1 above)
Comment (shultseva) [reply docs/modules/ROOT/pages/list-of-metrics.adoc:106]: I looked through the code and saw that there is no unified approach for the word cancel) so, here you recommend changing the parameter in the code?

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Pull request merged**